src/main/java/com/stripe/param/PaymentIntentCreateParams.java

```diff
@@ -196,6 +213,8 @@ public static Builder builder() {
 
     private Boolean confirm;
 
+    private ConfirmationMethod confirmationMethod;
```

Comment:
Because I had explicitly un-regen this confirmation method in 44c3448, it shows up in this last commit as an addition. Overall this PR, based on master, does nothing to this confirmation method.